packages/@aws-cdk/aws-ecs/lib/drain-hook/instance-drain-hook.ts

```diff
@@ -97,11 +97,21 @@ export class InstanceDrainHook extends cdk.Construct {
       actions: [
         'ecs:ListContainerInstances',
         'ecs:SubmitContainerStateChange',
-        'ecs:SubmitTaskStateChange',
-        'ecs:UpdateContainerInstancesState',
-        'ecs:ListTasks'
+        'ecs:SubmitTaskStateChange'
       ],
       resources: [props.cluster.clusterArn]
     }));
+
+    // Restrict to the container-instance operations to the ECS Cluster
```

Comment:
"Restrict ~to~ the container-instance operations to the ECS cluster"

Reply:
Addressed